app/services/framework_nomis_mappings/importer.rb

```diff
@@ -81,6 +82,7 @@ def nomis_code_ids_to_mappings
             end
           elsif mapping_nomis_fallback
             hash[mapping_nomis_fallback.id] = hash[mapping_nomis_fallback.id].to_a + [mapping]
+            log_exception('New NOMIS codes imported', [{ code: mapping.code, type: mapping_nomis_fallback.code_type }])
```

Comment:
Should we only warn for new codes?

Reply:
We might need to add custom config to pickup warnings in slack, so best stick to errors for now